mod_reforged/config/text/text.nut

```diff
@@ -0,0 +1,10 @@
+::Reforged.Text <- {
+    function getDayHourString( _seconds )
+    {
+        local days = ::Math.floor(_seconds / ::World.getTime().SecondsPerDay);
+        local hours = ::Math.floor(_seconds / ::World.getTime().SecondsPerHour);
+        local dayHourString = (days == 0) ? "" : (days == 1) ? "1 day and " : days + " days and ";
```

Comment:
Generally avoid nested ternaries. Enduriel and I have discussed this over at MSU and we are not in favor of it. In this particular case, it is still readable with the nested ternaries, so let's see what @Enduriel says.

Reply:
Yeah unfortunately as much as it's not too bad in this case I think we should avoid them completely. Really makes you wish for a match statement but without one just use a short switch probably for this unfortunately.